docs/quickstart.md

```diff
@@ -50,30 +48,31 @@ PostgreSQL database at localhost:5432.
 ```shell
 #!/usr/bin/env sh
 
-VERSION=4
+VERSION=5
 
-#Get the Gateway Jar
-curl https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/${VERSION}/gateway-ha-${VERSION}-jar-with-dependencies.jar -o ./gateway-ha.jar
-
-#Start the persistence DB
-if ! $(command -v psql >/dev/null); then
-    if ! $(command -v brew >/dev/null); then
-        echo 'Please install psql and rerun';
-        exit 1
-    else
-        brew install libpq
-    fi
+#Check and get the Gateway Jar
+if [[ -f "gateway-ha.jar" ]]; then
+    echo "Trino Gateway Jar already exists so skip fetching."
+else
+    echo "Trino Gateway Jar does not exist, try to fetch from maven repository."
+    curl https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/${VERSION}/gateway-ha-${VERSION}-jar-with-dependencies.jar -o ./gateway-ha.jar
 fi
 
-export PGPASSWORD=mysecretpassword
-docker run --name local-postgres -p 5432:5432 -e POSTGRES_PASSWORD=$PGPASSWORD -d postgres:latest
-#make sure the DB has time to initialize
-sleep 5
-
-#Initialize the DB
-curl https://raw.githubusercontent.com/trinodb/trino-gateway/main/gateway-ha/src/main/resources/gateway-ha-persistence-postgres.sql > ./gateway-ha-persistence-postgres.sql 
-psql -U postgres -h localhost -c 'CREATE DATABASE gateway'
-psql  -U postgres -h localhost -d gateway -f ./gateway-ha-persistence-postgres.sql 
+#Check if DB is running
+if docker ps --format '{{.Names}}' | grep -q '^local-postgres$'; then
+    echo "PostgreSQL database container 'localhost-postgre' is already running. Only starting Trino Gateway."
+else
+    echo "PostgreSQL database container 'localhost-postgre' is not running, initialize and run database"
+    export PGPASSWORD=mysecretpassword
+    docker run --name local-postgres -p 5432:5432 -e POSTGRES_PASSWORD=$PGPASSWORD -d postgres:latest
+    #Make sure the DB has time to initialize
+    sleep 5
+
+    #Initialize the DB
+    docker exec local-postgres psql -U postgres -h localhost -c 'CREATE DATABASE gateway'
+    docker cp ../gateway-ha/src/main/resources/gateway-ha-persistence-postgres.sql local-postgres:/tmp
```

Comment:
could you start the container with `docker run -v ../gateway-ha/src/main/resources/gateway-ha-persistence-postgres.sql:/tmp/gateway-ha-persistence-postgres.sql ...` instead of copying?

Reply:
changed to use mount volume command :)